packages/plugin-conversation/src/share-activity.js

```diff
@@ -316,7 +334,14 @@ ShareActivity.create = function create(conversation, object, spark) {
   });
 
   if (files) {
-    files.forEach((file) => share.add(file));
+    files.forEach((file) => {
+      if (Array.isArray(file)) {
```

Comment:
I am unsure what situation you are referring to that would fail, @ianwremmel.  My example might be from legacy `spark.conversation.share`.  Looks like the new one will take an object or an array of files.